ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,55 @@
+# global
+import numpy as np
+from hypothesis import given, strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+import ivy.functional.backends.numpy as ivy_np
+import ivy.functional.backends.torch as ivy_torch
+from ivy_tests.test_ivy.helpers import handle_cmd_line_args
+
+
+# result_type
+@handle_cmd_line_args

```

Comment:
Feel free to implement any other function from this task as this one is work in progress